### PR TITLE
optimize:optimize enumerated classes and add unit tests

### PR DESCRIPTION
**Pull request by MentosL**
### Ⅰ. Describe what this PR did
Optimize the code

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
Unit tests have been added

### Ⅳ. Describe how to verify it
Local calibration

### CODE REVIEW

**Comment by a364176773** [comment]
please standardize the content of the title and add relevant test cases

File: core/src/test/java/io/seata/core/model/GlobalStatusTest.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 public class GlobalStatusTest {
     private static final int BEGIN_CODE = 1;
     private static final int NONE = 99;
+    private static final int MIN_CODE = 0;
+    private static final int Max_CODE = 15;
```
Comment (l81893521) [inline core/src/test/java/io/seata/core/model/GlobalStatusTest.java:30]: MAX_CODE

Comment (MentosL) [reply core/src/test/java/io/seata/core/model/GlobalStatusTest.java:30]: OK   Now go fix it

**Review by l81893521 — APPROVED** [review]
LGTM

**Review by a364176773 — APPROVED** [review]
LGTM

**Pull request merged**